networkx/utils/backends.py

```diff
@@ -2259,13 +2287,13 @@ def _make_doc(self):
         """Generate the backends section at the end for functions having an alternate
         backend implementation(s) using the `backend_info` entry-point."""
 
-        if not self.backends:
+        if self.backends.issubset({"networkx"}):
```

Comment:
Is this related to `is_stub`?  
Can we just put it in a separate PR?

Why is it in this PR?

Reply:
This PR adds `"networkx"` to `self.backends`. This check means we don't need to add "Backends" section to the docstring if it is not implemented by any backends or is only implemented by "networkx".